src/components/CCPForm/CCPFormPage.tsx

```diff
@@ -112,24 +134,24 @@ const CCPFormPage = ({
     if (mode === 'edit') {
       editCCP({
         variables: {
-          id: ccpID,
+          id: ccpId,
           name: ccpName,
-          eventId: eventID,
+          eventId: eventId,
         },
       });
     } else {
       addCCP({
         variables: {
           name: ccpName,
-          eventId: eventID,
+          eventId: eventId,
           createdBy: 4,
```

Comment:
Just want to note, if you get an error when creating a new CCP, it's likely that the `createdBy` id is invalid. Update it to a valid id then try again :)